### PR TITLE
Don't use `-D` parameter with `install`.

### PR DESCRIPTION
**Pull request by propensive**

### CODE REVIEW

File: etc/fury [hunk 1]
```diff
@@ -91,8 +91,8 @@ ngServer() {
     SCALA=$(coursier fetch --classpath org.scala-lang:scala-reflect:2.12.8 com.facebook:nailgun-server:1.0.0)
     java -Dfury.home="${FURYHOME}" -cp "$SCALA:$CLASSPATH" "$MAIN" "$PORT" &
     NG_PID=$!
-    install -D <(echo $NG_PID) $PIDFILE
```
Comment (odisseus) [inline etc/fury:94]: How do you make sure that all the parent directories of `$PIDFILE` exist in the file system?

Comment (propensive) [reply etc/fury:94]: I ended up removing the use of a PID file completely.

File: etc/fury [hunk 2]
```diff
@@ -82,22 +82,26 @@ stopFury() {
   nailgun "${FURY_MAIN}" "stop" && killFury
 }
 
+alive() {
+  bash -c "(exec 6<> /dev/tcp/127.0.0.1/${PORT} > /dev/null 2> /dev/null)"
```
Comment (odisseus) [inline etc/fury:86]: I'm afraid this won't work in some situations. For example, what about the moment when the Nailgun process is already running, but hasn't started listening for connections yet? If the launcher script is run multiple times in quick succession (as happens during the integration tests), this situation is almost certain to happen.

Comment (odisseus) [reply etc/fury:86]: Nevermind, this turned out to work as intended.

**Pull request merged**